Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorWindow.cs

```diff
@@ -62,6 +82,7 @@ public static bool OpenAsset(int instanceId, int line)
             window.m_IsDirty = false;
             window.m_AssetId = instanceId;
             window.titleContent = new GUIContent("Input Actions Editor");
+            window.SetAsset(asset, actionToSelect, actionMapToSelect);
```

Comment:
There is duplicate `window.SetAsset()` call a couple of lines below, I assume this one should be moved down to replace the other one, so it's after `minSize` is set?

Reply:
Ah, good catch. It was a leftover from a merge conflict fix. Done ✅